src/components/reusable/pagetitle/pageTitle.ts

```diff
@@ -0,0 +1,90 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import PageTitleScss from './pageTitle.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+
+/**
+ * Page Title
+ */
+
+@customElement('kyn-page-title')
+export class PageTitle extends LitElement {
+  static override styles = PageTitleScss;
+
+  /** Whether to show headline. */
+  @property({ type: Boolean })
+  showheadLine = false;
+
+  /** Headline text. */
+  @property({ type: String })
+  headLine = '';
+
+  /** Page title text (required). */
+  @property({ type: String })
+  pageTitle = '';
+
+  /** Whether to show subtitle. */
+  @property({ type: Boolean })
+  showSubTitle = false;
```

Comment:
Same comment about "show" props not being needed.

Reply:
Done